app/styles/ui/_merge-status.scss

```diff
@@ -58,9 +58,16 @@
 dialog#merge {
   .merge-info {
     margin-top: var(--spacing-half);
+    margin-bottom: var(--spacing);
   }
 
   .merge-status-component {
-    margin-top: var(--spacing);
+    margin-top: 0;
+  }
+
+  .merge-status-icon-container {
+    &:after {
+      border-bottom: 0;
+    }
   }
```

Comment:
One last thing, just because the spacing is a little big now, after line 72 could you add: 

```
.dialog-footer {
    padding-top: var(--spacing);
}
```
I think that should fix it! After this we should be good to go. This looks awesome, thank you so much!

Reply:
@ampinsk beautiful!

I've updated the screencaps at original post to reflect the latest.